app/modes/modes.F90

```diff
@@ -152,10 +192,56 @@ program modes
         iMode = ModesToPlot(ii)
         write(lcTmp2,*)'Eigenmode',iMode,eigenValues(iMode)*Hartree__cm,'cm-1'
         call writeXYZFormat(lcTmp, geo%coords, geo%species, geo%speciesNames,&
-            & velocities=displ(:,:,ii), comment=trim(lcTmp2), append=isAppend)
+            & velocities=displ(:,:,iMode), comment=trim(lcTmp2), append=isAppend)
       end do
     end if
+  end if
+
+  write(stdout, *)'Vibrational modes'
+  if (allocated(bornMatrix) .and. allocated(bornDerivsMatrix)) then
+    write(stdout, "(T7,A,T16,A)")'freq.', 'polar.'
+    write(stdout, "(A,T7,A,T16,A)")'Mode', 'cm-1', 'arb.u.'
```

Comment:
Shouldn't `trans. dip` being included in the header here (and omitted further done)?

Reply:
fixed